src/pages/AllNewslettersPage.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import { Row, Col } from 'react-bootstrap'
+import styled from 'styled-components'
+import Header from '../components/Header'
+import Footer from '../sections/Footer'
+
+import { API_ROOT } from '../constants'
+
+
+const ReactMarkdown = require('react-markdown')
+const fs = require('fs')
+
+const Section = styled(Row)`
+    margin:50px;
+`
+
+
+class AllNewslettersPage extends Component {
+
+    state = {
+        markdownContent: null,
+    }
+
+
+    componentDidMount() {
+
+        fetch(`${API_ROOT}/fetchAllNewsletters`)
+        .then(response => {
+            if (response.ok) {
+                return Promise.resolve(response);
+            } else {
+                return Promise.reject(new Error(response.statusText));
+            }
+        })
+        .then(response => response.json())
+        .catch(error => {
```

Comment:
When it comes to `then/catch` statements, it's more typical to put the catch all the way at the bottom so please refactor to `.then().then().catch()` instead of `.then().catch().then()`

There are some rare cases where you want to stick a `catch` in the middle but no need here I don't think